### DISCUSSION TITLE
Incompatible with Arduino Giga R1 and Arduino Portenta boards

### DISCUSSION THREAD

**Discussion by L0rdS474n** [Bodmer/TFT_eSPI#2708] [New processor/display support or capability request]
Downloaded SquareLine Studio for building my GUI project.
Building the project works fine but when I try to compile it in Arduino IDE it does fire alot of SPI errors from this package.

If I switch to Dune board it works as it should but both Portenta and Giga boards are broken.

Is this known error or do you need the exact error messages?

**Comment by Bodmer** [comment]
I have not tested those boards but would expect them to work with a SPI display on the default SPI pins (D8, D9, D10 for Portenta H7) since the generic SPI calls are used.
